sdk/src/vm.ts

```diff
@@ -62,46 +68,68 @@ export class VM {
 
   public editor = {
     /**
-     * Open one of several files in tabs and/or split panes
+     * Open one of several files in tabs and/or split panes.
+     *
+     * @since 1.7.0 Added support for opening multiple files
      */
-    openFile: (path: OpenFilePath): Promise<null> => {
+    openFile: (path: OpenFileOption): Promise<null> => {
       return this._rdc.request({
         type: 'SDK_OPEN_FILE',
         payload: { path },
       });
     },
 
+    /**
+     * Set a project file as the currently selected file.
+     * 
+     * - This may update the highlighted file in the file explorer,
+     *   and the currently open and/or focused editor tab.
+     * - It will _not_ open a new editor tab if the provided path does not
+     *   match a currently open tab. See `vm.editor.openFile` to open files.
+     *
+     * @since 1.7.0
+     * @experimental
+     */
+    setCurrentFile: (path: string): Promise<null> => {
```

Comment:
New feature that is useful for multi-step demos with more than one file open at a time (e.g. 2 files in 2 side-by-side panes).

In a multi-step demo, it can be useful to explicitly set the currently focused file out of all the files that are open in tabs, so that:

1. That file gets revealed in the file explorer sidebar (e.g. expanding the folder hierarchy to reveal that file).
2. And to make sure that the file's tab gets the "open" state, if it's one of several tabs in a pane.

Reply:
Right now I'm erring on the side of _not_ opening an editor tab for the file if there isn't one already. Why? Because that would raise complex questions that we could only answer with questionable defaults.

Let's say there are open tabs (and maybe several panes) already:

- Do we keep the existing tabs and just open a new one? Hmm probably.
- Okay but what if at least one of the existing tabs is in "preview" mode (which will happen when using `openFile` to open one or several files initially): do we replace one of the tabs then? (Technically we should, but that might be an unexpected result!)
- And if we have several panes, where do we add the new tab? First pane? The pane which hosts the currently focused editor tab (if any)? Will that make the result unpredictable, depending on user actions?

In a multi-step demo, it's probably better for embedders to explicitly call:

```
// Reset the open tabs, even if that was the state used in the previous demo step,
// because the user may have interacted with the editor and closed some tabs!
await vm.editor.openFile(['src/foo.js', 'src/bar.js']);

// Select the right-side file
await vm.editor.setCurrentFile('src/bar.js');
```